src/waltz/mib/fd_netdev_netlink.c

```diff
@@ -168,7 +174,44 @@ fd_netdev_netlink_load_table( fd_netdev_tbl_join_t * tbl,
         }
         netdev->master_idx = (short)master_idx;
         break;
-      }
+      } /* IFLA_MASTER */
+
+      case IFLA_LINKINFO : {
+        if ( ifi->ifi_type!=ARPHRD_IPGRE ) continue;
+        
+        struct rtattr * info_rat    = rta;
+        long            info_rat_sz = (long)rta_sz;
+        
+        for( ; RTA_OK( info_rat, info_rat_sz ); info_rat = RTA_NEXT( info_rat, info_rat_sz ) ) {
+          void * info_rta    = RTA_DATA( info_rat );
+          ulong  info_rta_sz = RTA_PAYLOAD( info_rat ); 
+
+          switch( info_rat->rta_type ) {
+          case IFLA_INFO_KIND: {
+            char * kind = (char *)info_rta;
+            if( FD_UNLIKELY( strcmp( kind, "gre" ) !=0 ) ) FD_LOG_ERR(( "rta is not type gre: %s", kind ));
```

Comment:
Don't FD_LOG_ERR here, silently skip. (There might be an ARPHRD_IPGRE type device that's not actually a GRE tunnel)